### ISSUE TITLE
Replace deprecated v2.ToTensor() and unify transform style in origin_dataset.py

### ISSUE THREAD

**Issue by xyf5432** [ByteDance-Seed/VeOmni#1102]
### Summary

`veomni/data/diffusion/preprocess/origin_dataset.py` uses **two different transform styles** in the same file:

1. **Line 57** (`frame_process`) uses the v2 namespace with a **deprecated alias**:

```python
self.frame_process = v2.Compose(
    [
        v2.CenterCrop(size=(height, width)),
        v2.Resize(size=(height, width), antialias=True),
        v2.ToTensor(),          # ← v2 namespace, emits UserWarning on construction
        v2.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5]),
    ]
)
```

2. **Line 160** (`image_processor`) uses the v1 namespace:

```python
self.image_processor = transforms.Compose(
    [
        transforms.CenterCrop((height, width)) if center_crop else transforms.RandomCrop((height, width)),
        transforms.RandomHorizontalFlip() if random_flip else transforms.Lambda(lambda x: x),
        transforms.ToTensor(),
        transforms.Normalize([0.5], [0.5]),
    ]
)
```

`v2.ToTensor()` is a deprecated alias: constructing it emits a `UserWarning` (verified on torchvision 0.26.0):

> The transform `ToTensor()` is deprecated and will be removed in a future release. Instead, please use `v2.Compose([v2.ToImage(), v2.ToDtype(torch.float32, scale=True)])`.

This warning has been present since torchvision 0.15 (when the v2 namespace was introduced) — `torchvision/transforms/v2/_deprecated.py` has called `warnings.warn` in `ToTensor.__init__` since the first release.

### Proposed fix

**1. Replace the deprecated `v2.ToTensor()` (line 57)** with the officially recommended equivalent:

```python
self.frame_process = v2.Compose(
    [
        v2.CenterCrop(size=(height, width)),
        v2.Resize(size=(height, width), antialias=True),
        v2.ToImage(),
        v2.ToDtype(torch.float32, scale=True),
        v2.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5]),
    ]
)
```

**2. (Optional, style unification) Migrate the v1 pipeline (line 160) to the v2 namespace**, so the file uses a single consistent style. The v1 pipeline has v2 equivalents with the same semantics (`transforms.Compose` → `v2.Compose`, `transforms.CenterCrop` → `v2.CenterCrop`, `transforms.RandomHorizontalFlip` → `v2.RandomHorizontalFlip`, `transforms.ToTensor` → `v2.ToImage` + `v2.ToDtype`). Note the v1 pipeline itself does *not* emit any warning — the v1 namespace is fully supported — so this is a consistency improvement rather than a fix for a warning.

### Validation (torchvision 0.26.0+cpu)

- `v2.ToTensor()(img)` and `v2.Compose([v2.ToImage(), v2.ToDtype(torch.float32, scale=True)])(img)` produce identical output up to float precision: max abs diff = 5.96e-8 (a single last-bit rounding difference on uint8→float32 scaling; the official deprecation message itself states "Output is equivalent up to float precision").
- With `warnings.simplefilter("error", UserWarning)`: the current line-57 pipeline raises the deprecation `UserWarning`; the proposed pipeline constructs and runs cleanly.

### Impact

- No deprecation `UserWarning` during training; no behavior change to data preprocessing; the file ends up with one consistent transform style.

### References

- [torchvision docs: v2.ToTensor (deprecated)](https://pytorch.org/vision/stable/generated/torchvision.transforms.v2.ToTensor.html) — "[DEPRECATED] Use `v2.Compose([v2.ToImage(), v2.ToDtype(torch.float32, scale=True)])` instead."
- torchvision source `torchvision/transforms/v2/_deprecated.py` — `warnings.warn` in `ToTensor.__init__`, present since 0.15.
